ci-operator/step-registry/ipi/conf/ipi-conf-ref.yaml

```diff
@@ -0,0 +1,10 @@
+ref:
+  as: ipi-conf
+  from: centos:7
+  commands: ipi-conf-commands.sh
+  resources:
+    requests:
+      cpu: 1000m
+      memory: 2Gi
```

Comment:
this memory limit is shocking ;).  I *hope* we aren't using that much memory to generate an install-config.  But I guess we can always refine this later, certainly doesn't need to block this PR.

Reply:
Copy-paste, thanks for catching it.